src/com/nutrons/nu17/RobotMap.java

```diff
@@ -2,41 +2,22 @@
 
 public class RobotMap {
 
```

Comment:
The bit that said they were wheels was helpful.  Add a comment:
// Wheel motors